.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: push
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ contains(fromJSON('["2.5.0", "ruby-head", "jruby-9.1", "jruby-head"]'), matrix.ruby-version) }}
+    strategy:
+      matrix:
+        ruby-version:
+          - 2.5.0
+          - 2.7.2
+          - 3.0.0
+          - ruby-head
+          - jruby-9.1.17
```

Comment:
Have to pump this to `jruby-9.1.17` since `ubuntu-latest` doesn't have `jruby-9.1.16.0`. See [this](https://github.com/QQism/graphlient/runs/6617244832?check_suite_focus=true)